docs/src/main/tut/README.md

```diff
@@ -37,7 +37,7 @@ import freestyle._
 <div class="row" markdown="1">
 <div class="col-md-6 col-md-push-6" markdown="1">
 # Automatic Dependency Injection
-Freestyle includes all the implicit machinery necessary to achieve seamless dependency injection of [`@free`](/docs/core/algebras) and [`@module`](/docs/core/modules) Algebras.
+Freestyle includes all the implicit machinery necessary to achieve seamless dependency injection of [`@free`](/docs/src/main/tut/docs/core/algebras) and [`@module`](/docs/src/main/tut/docs/core/modules) Algebras.
```

Comment:
These are internal links in frees.io also. To get them to work in github and inside frees.io we may just use the absolute urls in frees.io to whatever content they are pointing to.